service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToSubmitTest.java

```diff
@@ -198,8 +197,6 @@ void shouldPopulateStandardDirectionOrderObjectFromUploadRoute() {
 
     @Test
     void shouldUpdateStateWhenOrderIsSealedThroughServiceRouteAndRemoveRouterAndSendNoticeOfProceedings() {
-        given(featureToggleService.isSendNoticeOfProceedingsFromSdo()).willReturn(true);
```

Comment:
same here

    @MockBean
    private FeatureToggleService featureToggleService;

to be reomved

Reply:
and
    @Autowired
    private NoticeOfProceedingsTemplateDataGenerationService noticeOfProceedingsTemplateDataGenerationService;